examples/webhook-signing/README.md

```diff
@@ -0,0 +1,54 @@
+# Checking webhook signatures
+
+Verify the events that Stripe sends to your webhook endpoints. Additional details in the Stripe [docs](https://stripe.com/docs/webhooks/signatures).
+
+You can find a Node.js TypeScript server example in [stripe-samples](https://github.com/stripe-samples/accept-a-card-payment/tree/master/using-webhooks/server/node-typescript).
+
+### Requirements
+
+You’ll need the following:
+
+- [Node.js](http://nodejs.org) >=10.0.0
+- Stripe account to accept payments ([sign up](https://dashboard.stripe.com/register) for free).
+- [Stripe CLI](https://github.com/stripe/stripe-cli) or [ngrok](https://ngrok.com/) to tunnel requests to your local server.
+
+### Setup
+
+Copy the environment variables file from the root of the repository:
+
+    cp .env.example .env
+
+Update `.env` with your own [Stripe API keys](https://dashboard.stripe.com/account/apikeys).
+
+### Install and run
+
+Install dependencies using npm:
```

Comment:
```suggestion
Install dependencies:
```

(don't want to reply that `yarn` won't work for some reason)

Reply:
We should probably specify what directory all of these commands should be run in

Reply:
added to first setup sentence